rabbitmq/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM rabbitmq:latest
```

Comment:
not 100% sure, but i think we should have a fixed version here,
unless we have automated smoke tests.
Otherwise, it might stop working from one rollout to the next

Reply:
Good point.
@teosarca any suggestions about a specific version?
`rabbitmq:3.7.4` seems to be the latest release according to https://www.rabbitmq.com/download.html

Reply:
@metas-jb  3.7.4 it's perfect.